src/metabase/notification/payload/execute.clj

```diff
@@ -180,7 +185,7 @@
   (cond
     (:card_id dashcard)
     (let [parameters (merge-default-values parameters)]
-      (execute-dashboard-subscription-card dashcard parameters))
+      (update (execute-dashboard-subscription-card dashcard parameters) :result data-rows-to-disk!))
```

Comment:
only do this for dashboard subscription but not alerts since alert.

Reply:
Let's add a corresponding comment to the alert code, so that it's not forgotten.